lib/media/track/videotrack.js

```diff
@@ -185,6 +404,17 @@ function dimensionsChanged(track, elem) {
  *   {@link VideoTrack} has not yet started
  */
 
+/**
+ * A {@link VideoProcessor} is used to process incoming video frames before
```

Comment:
Can you add something about `processFrame` behavior:
* Any exception raised (either synchronously or asynchronously) in `processFrame` will result in the frame being dropped.
* Developers can drop the current frame by returning null.